src/code/z_collision_btltbls.c

```diff
@@ -790,7 +790,7 @@ DamageTable sDamageTablePresets[] = {
  * is out of range.
  */
 DamageTable* DamageTable_Get(s32 index) {
-    if (index < 0 || index > ARRAY_COUNT(sDamageTablePresets) - 1) {
+    if ((index < 0) || (index > (ARRAY_COUNT(sDamageTablePresets) - 1))) {
```

Comment:
Now that you are here
```suggestion
    if ((index < 0) || (index >= ARRAY_COUNT(sDamageTablePresets))) {
```